docs/operations-guide/running-metabase-on-azure.md

```diff
@@ -0,0 +1,183 @@
+- Publish: Docker Container
+- Operating System: Linux
+- Region: same as the one on the previous steps
+- App Service Plan: if you don't have one already, it will create a new one automatically
+- SKU and Size: change it to a Production level plan with **AT LEAST** 200 total ACU and 3.5GB of memory, click **Apply** after choosing the specs
+
+Now go to the next step where you will select:
+- Options: single container
+- Image source: DockerHub
+- Access Type: Public
+- Image and tag: metabase/metabase:latest (or choose any other docker image tag of your preference, like our Enterprise Edition that has super-powers). If you want to see which is the latest Metabase version you can check in our [Community Edition Dockerhub repository](https://hub.docker.com/r/metabase/metabase/tags?page=1&ordering=last_updated) and also our [Enterprise Edition Dockerhub Repository](https://hub.docker.com/r/metabase/metabase-enterprise/tags?page=1&ordering=last_updated)
+- Startup command: empty
+
+Click next till you get to the last section and click on **Create** to initialize the creation of your application and wait till it's over.
+
+Now go to te application configuration page and click on Settings --> Networking on the left side of the page, where a page will appear and you will have to click on **Click here to configure** under **VNET integration**.
+
+![Azure VNET integration](images/AZVNETintegration.png)
+
+Now click on the huge plus sign next to **Add VNET** and select the VNET that you created on the first step and the public subnet to set the server on the public subnet of the VNET, and click OK.
+
+![Azure VNET public subnet](images/AZVNETPublicSubnet.png)
+
+Now go back to te application configuration page and click on Settings --> Configuration on the left side of the page where you will see a few Application Settings already configure.
+
+Here you will need to add the [Enviroment Variables]() for connecting Metabase to its [PostgreSQL Application Database](), but make sure that you use the **MB_DB_CONNECTION_URI** as PostgreSQL in Azure is configured by default with SSL so currently the only way of passing those parameters is by using the full URI.
+
+Also, take into account that the username in Azure PostgreSQL is `user@name_of_your_database_engine` so in this case the entire connection uri would be as follows: `postgresql://databasePrivateIPAddress:port/postgres?user=user@name_of_your_database_engine&password=configuredpassword&ssl=true&sslmode=required`.
+
+Lets put an example, if you got this information from the previous steps:
+1) database private IP address: 10.0.2.4
+2) database port: 5432 (in the case of Postgres, MySQL/MariaDB default port is 3306)
+3) name of the database server: metabase-app-database
+4) username of the database: metabase
+5) password: Password1!
+
+then my connection string will be: `postgresql://10.0.2.4:5432/postgres?user=metabase@metabase-app-database&password=Password1!&ssl=true&sslmode=require`
```

Comment:
```suggestion
Then your connection string would be: 
```
postgresql://10.0.2.4:5432/postgres?user=metabase@metabase-app-database&password=Password1!&ssl=true&sslmode=require
```
```

Reply:
The connection string should be on it's own line.